File: DailyFlash Solutions/DailyFlash Solutions/55-DailyFlash_Solutions/24_Mar_Solution_Two/Java/Program0.java
import java.io.*;

class Program0 {


	public static void main(String[] args) throws IOException {

		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		float sum = 0;
		int l = 0;
		System.out.printf("Enter The Length of the Series\n");

		l = Integer.parseInt(br.readLine());


		for(int i = 0; i < l; i++)
			sum+=(i*3.14f)/6;
		System.out.println("Sum of Series = "+String.format("%.2f",(float)sum));
	}
}

File: DailyFlash Solutions/DailyFlash Solutions/55-DailyFlash_Solutions/24_Mar_Solution_Two/CPP/Program0.cpp
#include <stdio.h>

#define PI 3.14

int main() {

	int l = 0;
	float sum = 0;
	printf("Enter The Length of the Series\n");
	scanf("%d",&l);


	for(int i = 0; i < l; i++)
		sum+=(i*PI)/6;

	printf("Sum of Series = %.2f\n",sum);

	return 0;
}
